module/VuFindSearch/src/VuFindSearch/Command/CallMethodCommand.php

```diff
@@ -110,8 +128,12 @@ public function execute(BackendInterface $backend): CommandInterface
             );
         }
         $args = $this->getArguments();
-        return $this->finalizeExecution(
+        $this->finalizeExecution(
```

Comment:
Should you add a call to `resetExtraRequestDetails` prior to finalizing execution, to be sure that no values from a previous command bleed into this one?

Reply:
For now it would not do much, but i guess it makes sense for consistency reasons.